### YamlMime:UniversalReference
api_name: []
items:
- children:
  - azure.cognitiveservices.vision.computervision.models.computer_vision_error_py3.ComputerVisionError
  - azure.cognitiveservices.vision.computervision.models.computer_vision_error_py3.ComputerVisionErrorException
  fullName: azure.cognitiveservices.vision.computervision.models.computer_vision_error_py3
  langs:
  - python
  module: azure.cognitiveservices.vision.computervision.models.computer_vision_error_py3
  name: computer_vision_error_py3
  source:
    id: computer_vision_error_py3
    path: azure-cognitiveservices-vision-computervision\azure\cognitiveservices\vision\computervision\models\computer_vision_error_py3.py
    remote:
      branch: master
      path: azure-cognitiveservices-vision-computervision\azure\cognitiveservices\vision\computervision\models\computer_vision_error_py3.py
      repo: https://github.com/Azure/azure-sdk-for-python.git
    startLine: 0
  type: module
  uid: azure.cognitiveservices.vision.computervision.models.computer_vision_error_py3
references:
- fullName: azure.cognitiveservices.vision.computervision.models.computer_vision_error_py3.ComputerVisionError
  isExternal: false
  name: ComputerVisionError
  parent: azure.cognitiveservices.vision.computervision.models.computer_vision_error_py3
  uid: azure.cognitiveservices.vision.computervision.models.computer_vision_error_py3.ComputerVisionError
- fullName: azure.cognitiveservices.vision.computervision.models.computer_vision_error_py3.ComputerVisionErrorException
  isExternal: false
  name: ComputerVisionErrorException
  parent: azure.cognitiveservices.vision.computervision.models.computer_vision_error_py3
  uid: azure.cognitiveservices.vision.computervision.models.computer_vision_error_py3.ComputerVisionErrorException
